### ISSUE TITLE
Consider adding graphql-scalars package

### ISSUE THREAD

**Issue by johnymontana** [neo4j-graphql/neo4j-graphql-js#347]
The [graphql-scalars](https://www.npmjs.com/package/graphql-scalars) package makes available a number of common scalars and includes validation.

We should consider adding any relevant scalar type definitions and resolvers from this library to make available to the user as a convenience.

Of course there is the potential for conflicts with user defined custom scalars so that must be taken into account.

**Comment by benjamin-rood** [comment]
Is the intent to integrate the package somehow, i.e. no need to define the types or resolvers manually? Because as far as I can tell, there isn't an impediment to anyone using `graphql-scalars` in an existing Node GraphQL server using `neo4j-graphql-js`.

**Comment by johnymontana** [comment]
Yeah, my thought was to pull in the relevant types and resolvers so they are available by default. 

There might be some additional work to make sure features like filtering are supported as well.

**Comment by michaeldgraham** [comment]
https://github.com/neo4j-graphql/neo4j-graphql-js/issues/608
